tests/sync/test_medicaid_eligibility_api.py

```
@@ -0,0 +1,753 @@
+    def test_young_child_not_asked_about_pregnancy(self):
+        _, _, _, _, missing, _ = is_eligible(state="wa", age=2)
+        self.assertFalse(any("pregnant" in q.lower() for q in missing))
+
+    def test_unrecognized_state_becomes_a_reask_not_an_exception(self):
+        # A garbled state from the LLM used to raise ValueError and kill the
+        # whole check; now it just re-asks for the state.
+        _, _, _, _, missing, _ = is_eligible(state="medi-cal er california", age=30)
+        self.assertTrue(any("state" in q.lower() for q in missing))
+
+    def test_young_child_verdict_does_not_stall_on_pregnancy(self):
+        eligible_2025, _, _, _, missing, _ = is_eligible(
```

Ruff (0.16.1)

[warning] Unpacked variable `missing` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)